lib/prometheus_exporter/utils/sql_sanitizer.rb

```diff
@@ -0,0 +1,109 @@
+require 'prometheus_exporter/utils/rails'
+require 'active_support/core_ext/string/filters'
+
+module PrometheusExporter
+  module Utils
+    class SqlSanitizer
```

Comment:
I think we need a bunch of tests for the sanitizer it also would help clarify how it works

Reply:
I didn't want to reinvent the wheel so the code from the sanitizers was taken from the Scout App apm gem. They have tests for the sanitizers.

But, looking into their repo, I just realize reading the License file that their code is not open source :( So I don't know how to proceed.